test/e2e/lib/components/guide-component.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { By, Key } from 'selenium-webdriver';
+
+/**
+ * Internal dependencies
+ */
+import * as driverHelper from '../driver-helper.js';
+
+import AsyncBaseContainer from '../async-base-container.js';
+
+export default class GuideComponent extends AsyncBaseContainer {
+	constructor( driver ) {
+		super( driver, By.css( '.components-guide' ) );
```

Comment:
Does this need to match the `selector` param passed in the `dismiss` function?

Reply:
Nope, the dynamic selector is there just to increase specificity.